### PR TITLE
added tabs with content on badge

### PR DESCRIPTION
**Pull request by DecentralizedDan**
* updated index.html
* added badge.js to control tab behavior
* updated style.css to support tabs and content

Screenshot 2022-11-30 at 3 28 34 PM
Screenshot 2022-11-30 at 3 28 26 PM
Screenshot 2022-11-30 at 3 28 20 PM

After clicking the toggle:

Screenshot 2022-11-30 at 3 28 56 PM

### CODE REVIEW

File: source/static/style.css [hunk 1]
```diff
@@ -30,18 +29,154 @@ h1 {
...
+.dash-section {
+	padding-bottom: 2rem;
+}
+.dash-container {
+	margin-top: -1rem;
+	padding-bottom: 3.5rem;
+}
+.dash-row {
+	clear: both;
+	font-family: 'THICCBOI';
```
Comment (jmercouris) [inline source/static/style.css:60]: I can't not laugh when seeing this font name.

File: source/index.html [hunk 2]
```diff
@@ -6,25 +6,93 @@
...
+                  <input type="checkbox" id="switch" /><label for="switch">Toggle</label>
+                </div>
+                <div class="toggle-label" id="not-yet">
+                  Not yet
+                </div>
+              </div>
+                <p>We will mix your anonymized data with data from tens of thousands of other users, and sell that 
+              data stream to tech companies so they can make themselves more secure.</p>
+              <p>This generates revenue for Interlock to keep improving ThreatSlayer and our other security products. 
+                  It also helps tech companies secure themselves -- and all their users -- from new threats on the internet.</p>
```
Comment (jmercouris) [inline source/index.html:87]: I believe "Internet" should be capitalized since it is a proper noun

**Comment by jmercouris** [comment]
Thank you Dan, this looks *way* better. I really like it!

Interesting approach to use the div with dash item instead of styling a `ul` and `li`. I guess it doesn't truly matter since our page is not meant to be parsed by machines/semantically understood, however it does feel a bit odd :-)

File: source/static/style.css (hunk 1 above)
Comment (DecentralizedDan) [reply source/static/style.css:60]: in all caps no less, it is truly the best font name

**Comment by jmercouris** [comment]
Thank you Dan!

**Pull request merged**